exercises/armstrong-numbers/canonical-data.json

```diff
@@ -0,0 +1,189 @@
+{
+  "exercise": "armstrong-number",
+  "version": "1.0.0",
+  "comments": [
+    "Both input-parameters are always given in base-10(decimal)."
```

Comment:
I think "base-10(decimal)" should be "base-10 (decimal)"

Reply:
Missing space added